spec/support/global_state_reset.rb

```diff
@@ -0,0 +1,22 @@
+module OpenProject
+  ##
+  # Provides resets to any global state that may have been changed through tests
+  # and the change of which should not affect other tests.
+  module GlobalStateReset
+    module_function
+
+    def reset_global_state!
+      reset_current_user!
+    end
+
+    def reset_current_user!
+      User.current = User.anonymous
+    end
+  end
+end
+
+RSpec.configure do |config|
+  config.after(:suite) do
+    OpenProject::GlobalStateReset.reset_global_state!
```

Comment:
@ulferts Ah right. ^^ Didn't see that. In that case I'll just drop the commit altogether.